Reviews core terms (#1794)
temporalio/documentation: docs/php/versioning.md

return $result2;
```

> When `Workflow::getVersion()` is run for the new Workflow Execution, it records a marker in the Workflow
> history so that all future calls to `GetVersion` for this change Id--`Step 1` in the example--on this
> Workflow Execution will always return the given version number, which is `1` in the example.

Reviewer: ```suggestion
When `Workflow::getVersion()` is run for the new Workflow Execution, it records a marker in the Workflow history so that all future calls to `GetVersion` for this change Id—`Step 1` in the example—on this Workflow Execution will always return the given version number, which is `1` in the example.
```

`GetVersion()` call before it was introduced will return `DEFAULT_VERSION`. A Workflow that was run
with `maxSupported` set to 1, will return 1. New Workflows will return 2.

> After you are sure that all of the Workflow Executions prior to version 1 have completed, you can
> remove the code for that version. It should now look like the following:

Reviewer: ```suggestion
After you are sure that all of the Workflow Executions prior to version 1 have completed, you can remove the code for that version.
It should now look like the following:
```

> You'll note that `minSupported` has changed from `DEFAULT_VERSION` to `1`. If an older version of the
> Workflow Execution history is replayed on this code, it will fail because the minimum expected version

Reviewer: ```suggestion
You'll note that `minSupported` has changed from `DEFAULT_VERSION` to `1`.
If an older version of the Workflow Execution history is replayed on this code, it fails because the minimum expected version is 1.
```

> is 1. After you are sure that all of the Workflow Executions for version 1 have completed, then you
> can remove 1 so that your code would look like the following:

Reviewer: ```suggestion
After you are sure that all Workflow Executions for version 1 have completed, you can remove version 1 so that your code looks like the following:
```

```

> Upgrading a Workflow is straightforward if you don't need to preserve your currently running

Reviewer: ```suggestion
Upgrading a Workflow is straightforward if you don't need to preserve your currently running Workflow Executions.
```

> Workflow Executions. You can simply terminate all of the currently running Workflow Executions and
> suspend new ones from being created while you deploy the new version of your Workflow code, which does
> not use `GetVersion()`, and then resume Workflow creation. However, that is often not the case, and

Reviewer: ```suggestion
You can simply terminate all of the currently running Workflow Executions and suspend new ones from being created while you deploy the new version of your Workflow code, which does not use `GetVersion()`, and then resume Workflow creation.
```

> you need to take care of the currently running Workflow Executions, so using `GetVersion()` to update
> your code is the method to use.

Reviewer: ```suggestion
However, that is often not the case, and you need to take care of the currently running Workflow Executions, so using `GetVersion()` to update your code is the method to use.
```
